src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main>
+      <ul class="calendar calendar__month-length--29">
+        <li class="calendar__item calendar__item--1 calendar__item--start-day"></li>
```

Comment:
The start-day modifier should be added where the previous month-lenght was added, and it should have values like start-day--mon ... up until start-day--sun